cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/config/PdfPageLayout.java

```diff
@@ -16,7 +17,10 @@
     private float marginTop;
     private float marginBottom;
     private float lineDistance;
+    private float header;
     private float footer;
+    private ReportFooterOnPages footerOnPages;
+    private ReportFooterOnPages headerOnPages;
```

Comment:
I dont like those variable names, it is confusing, I dont know what it should be